sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md

```diff
@@ -20,7 +20,7 @@
 
 ### Features Added
 
-- Added the `QuestionAnsweringProjectsClient` to manage Question Answering projects.
+- Added the `QuestionAnsweringAuthoringClient` to manage Question Answering projects.
```

Comment:
should this rename be under 1.1.0-beta.3?

Reply:
Good catch. Too broad with my find/replace-all.